demo/src/main/java/iamutkarshtiwari/github/io/imageeditorsample/MainActivity.java

```diff
@@ -151,6 +151,8 @@ private void editImageClick() {
                     .withBrightnessFeature()
                     .withSaturationFeature()
                     .withBeautyFeature()
+//                    .withStickerFeature()
```

Comment:
Why are we commenting these?

Reply:
removed comment. its adds sticker feature in editor